test/module/shared_model/backend_proto/proto_query_response_factory_test.cpp

```diff
@@ -122,16 +122,23 @@ TEST_F(ProtoQueryResponseFactoryTest, CreateAccountDetailResponse) {
   const HashType kQueryHash{"my_super_hash"};
 
   const DetailType account_details = "{ fav_meme : doge }";
+  const size_t total_number = 999;
+  const boost::optional<AccountIdType> next_writer("pepe@uganda");
+  const boost::optional<AccountDetailKeyType> next_key = boost::none;
   auto query_response = response_factory->createAccountDetailResponse(
-      account_details, kQueryHash);
+      account_details, total_number, next_writer, next_key, kQueryHash);
```

Comment:
```suggestion
      account_details, total_number, next_record_id, kQueryHash);
```
Please fix the tests